assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_report_submissions.vm

```diff
@@ -202,8 +202,8 @@
 											$tlang.getString("gen.nograd")
 										#else
 											#if ($!assignment.IsGroup)
-												#if ($!submission.GradeForUser($!submission.User.Id))
-													$!submission.GradeForUser($!submission.User.Id)
+												#if ($!submission.getGradeForUser($!submission.User.Id))
+													$!service.getGradeDisplay($!submission.getGradeForUser($!submission.User.Id), $typeOfGrade, $!assignment.ScaleFactor)
```

Comment:
Instead of calling service.getGradeDisplay() in the template this could be handled in the SubmitterSubmission class.